python/lsst/ts/standardscripts/auxtel/daytime_checkout/latiss_checkout.py

```diff
@@ -0,0 +1,152 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["LatissCheckout"]
+
+import asyncio
+
+import yaml
+from astropy.time import Time
+from lsst.ts import salobj
+from lsst.ts.observatory.control.auxtel.latiss import LATISS
+
+
+class LatissCheckout(salobj.BaseScript):
+    """DayTime LATISS Checkout SAL Script.
+
+    This script performs the daytime checkout of
+    LATISS to ensure it is ready to be released
+    for nighttime operations
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    - "Bias Frame Verification": Before taking bias frame.
+    - "Engineering Frame Verification": Before taking engineering frame. Final
+    checkpoint in script.
+
+    **Details**
+
+    This script is used to perform the daytime checkout of LATISS to ensure it
+    is ready for nighttime operations. It does not include telescope or dome
+    motion. It will enable LATISS, take a bias and an engineering frame, and
+    check that both frames were successfully ingested by OODS.
+    """
+
+    def __init__(self, index=1):
+
+        super().__init__(
+            index=index,
+            descr="This script performs the daytime checkout of LATISS to ensure"
+            "it is ready to be released for nighttime operations.",
+        )
+
+        # Instantiate latiss. We need to do this after the call to
+        # super().__init__() above. We can also pass in the script domain and
+        # logger to both classes so log messages generated internally are
+        # published to the efd.
+        self.latiss = LATISS(domain=self.domain, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/auxtel/latiss_checkout.yaml
+            title: LatissDayTimeCheckout v1
+            description: LATISS Configuration for DayTimeCheckout.
+            type: object
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+
+        self.config = config
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = 5.0 * 60  # Approximate 5min to completion.
+
+    async def run(self):
+
+        # Bias Verification, start with checkpoint for observer
+        await self.checkpoint("Bias Frame Verification")
+
+        self.latiss.rem.atoods.evt_imageInOODS.flush()
+        await self.latiss.take_bias(nbias=1)
+        ingest_event = await self.latiss.rem.atoods.evt_imageInOODS.next(
+            flush=False, timeout=10
```

Comment:
Consider creating a constant for the timeout.

```
STD_TIMEOUT = 10
```

before the class definition (e.g. line 30), and using it here and elsewhere

Reply:
Change adopted and applied to other scripts.